.gitignore

```diff
@@ -1,3 +1,4 @@
 dist/
 node_modules/
 npm-debug.log
+.idea/
```

Comment:
Your IDE specific folders should not be in project `.gitignore`. Instead, you can add them to your personal `.git/info/exclude` file.